docs/_docs/Instructors/04-BLU-Office-Hours.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: BLU Office Hours
 category: Instructors
-order: 2
+order: 4
 ---
 
```

Comment:
Should we remove/refactor this page?

Reply:
halfway through batch 5, we tried to bring back office hours and students really liked it. I would keep it and push, alongside ops, for them to happen on batch 6 consistently. :)

Reply:
Will be done in a different MR